Tests/SwiftSyntaxTest/SyntaxTests.swift

```diff
@@ -144,4 +144,57 @@ public class SyntaxTests: XCTestCase {
     XCTAssertEqual(Syntax(integerExpr), Syntax(fromProtocol: integerExpr as SyntaxProtocol))
     XCTAssertEqual(Syntax(integerExpr), Syntax(fromProtocol: integerExpr as ExprSyntaxProtocol))
   }
+
+  public func testNumericLiteralSetters() {
+    var integerExpr = IntegerLiteralExprSyntax {
+      $0.useDigits(SyntaxFactory.makeIntegerLiteral("1"))
+    }!
+    XCTAssert(integerExpr.isValid)
+
+    var token: TokenSyntax
+    var source = "2"
+    var tree = try! SyntaxParser.parse(source: source)
+    token = tree.firstToken!
+    try! integerExpr.setDigits(token)
+    XCTAssert(integerExpr.isValid)
+
+    let invalidNumericLiteral = "🥥"
+
+    source = invalidNumericLiteral
+    tree = try! SyntaxParser.parse(source: source)
+    token = tree.firstToken!
+    do {
+      try integerExpr.setDigits(token)
```

Comment:
I think there’s `XCTAssertThrowsError` to test that an error is thrown. This test would also pass if `setDigits` doesn’t throw.

Reply:
Done.